request.go

```diff
@@ -31,7 +31,7 @@ func (c *Client) RawRequest(verb, p string, ro *RequestOptions) (*http.Request,
 
 	// Append the path to the URL.
 	u := *c.url
-	u.Path = path.Join(c.url.Path, p)
+	u.Path = strings.TrimRight(c.url.Path, "/") + "/" + strings.TrimLeft(p, "/")
```

Comment:
As far as I can tell, this preserves the current behavior. If I read your issue correctly, you don't want to alter `p` at all, so I think this needs to become:

```golang
u.Path = strings.TrimRight(c.url.Path, "/") + "/" + p
```